pandas/core/groupby/generic.py

```diff
@@ -1001,14 +1001,42 @@ def take(
         result = self._op_via_apply("take", indices=indices, axis=axis, **kwargs)
         return result
 
-    @doc(Series.skew.__doc__)
     def skew(
         self,
         axis: Axis | lib.NoDefault = lib.no_default,
         skipna: bool = True,
         numeric_only: bool = False,
         **kwargs,
     ) -> Series:
+        """
+        Return unbiased skew within groups.\n\nNormalized by N-1.
+
+        Parameters
+        ----------
+        axis : {axis_descr}
+            Axis for the function to be applied on.
+            For `Series` this parameter is unused and defaults to 0.
+
+            For DataFrames, specifying ``axis=None`` will apply the aggregation
+            across both axes.
+
+            .. versionadded:: 2.0.0
+
+        skipna : bool, default True
+            Exclude NA/null values when computing the result.
+        numeric_only : bool, default False
+            Include only float, int, boolean columns. Not implemented for Series.
+
+        {min_count}\
+        **kwargs
+            Additional keyword arguments to be passed to the function.
+
+        Returns
+        -------
+        {name1} or scalar\
+        {see_also}\
+        {examples}
```

Comment:
Similar to the above, this docstring isn't using a template so these need to be fleshed out. For the `See Also`, I would recommend `Series.skew`. For the examples, it'd be great to have some examples using groupby.